src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {

```

Comment:
Don't begin class or method implementation with an empty line. Please remove the redundant empty line before the class declaration.